Automattic/newspack-popups
# Clean up release zip

### All Submissions:






### Changes proposed in this Pull Request:



This gets rid of all unnecessary files from the release zip.

### How to test the changes in this Pull Request:

1. Create a release zip: `$ npm run build && npm run release:archive`
2. Inspect zip contents - it should only contain the license, PHP files, and `dist` folder with JS/CSS assets 
3. Upload the plugin to a new site, it should work as expected

### Other information:
